docs/en/03-benchmark/benchmark.md

```diff
@@ -1641,6 +1641,19 @@ Users can directly test the performance through [how_to_evaluate_a_model.md](../
     <td align="center">-</td>
     <td align="center">-</td>
   </tr>
+  <tr>
+    <td align="center"><a href="https://github.com/open-mmlab/mmrotate/blob/main/configs/roi_trans/roi_trans_r50_fpn_1x_dota_le90.py">RoI Transformer</a></td>
```

Comment:
Hi, could you add the new model config in regression test yml for easy test?  https://github.com/open-mmlab/mmdeploy/blob/69c38b9f74be84084f4b8ee357d9ebfeefb3c69f/tests/regression/mmrotate.yml#L43